src/jdk.jdwp.agent/share/native/libjdwp/log_messages.c

```diff
@@ -78,11 +78,12 @@ get_time_stamp(char *tbuf, size_t ltbuf)
                 "%d.%m.%Y %T", localtime(&t));
     (void)strftime(timestamp_timezone, TZ_SIZE,
                 "%Z", localtime(&t));
+
     // Truncate milliseconds in buffer large enough to hold the
-    // value which is always < 1000
+    // value which is always < 1000 (and so a maximum of 3 digits for "%.3s")
```

Comment:
With the updated code the comment is no longer correct - change %.3s to %.3d

Reply:
Ok.